lib/resources/parse_config.rb

```diff
@@ -52,21 +63,18 @@ def parse_file(conf_path)
         return skip_resource "Can't read file \"#{conf_path}\""
       end
 
-      read_content
+      read_params
     end
 
     def read_file(path)
       @files_contents[path] ||= inspec.file(path).content
     end
 
-    def read_content
-      # parse the file
-      @params = SimpleConfig.new(@content, @opts).params
-      @content
-    end
-
-    def to_s
-      "Parse Config #{@conf_path}"
+    def read_params
+      return @params if defined?(@params)
```

Comment:
@params.nil? would be the more idiomatic guard here I think.  Instance variables will return nil default if they haven't previously been defined. In our other ruby code bases I think we would generally follow this style:

```
def read_params
  @params ||= if content.nil?
                {}
              else
                SimpleConfig.new(content, @opts).params
              end
end
```

preferring the early return in cases where the assignment logic is more complex.  Although, I'm not sure what latest rubocop says about this.